javascript/javascript2/week3/delayCallApp.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+        function logOut(delay, stringToLog) {
```

Comment:
good job! But please make the correct the indentation, it's not very pleasant to read code with the wrong format

Reply:
I have downloaded Prettier,.now my codes are well indented, Thanks